### PR TITLE
Sentry SDK & Debugging Failures

### PR DESCRIPTION
**Pull request by thinkocapo**
- instrumented sentry sdk to capture errors from a few places

## debug
compare several things
1. events produced by the demo apps (these are expected)
2. events produced by test-data-automation (now instrumented with `sentry_sdk`, some of these are health checks, others are failtures we're okay with, others are failtures representing real failures). these events go to project `python-test-data-automation`
3. Metric Alerts on project `python-test-data-automation` for Low Traffic / Low Volume
4. can look at the `nohup.out` file.
5. Look at Sauce Labs VM's and the failed nodes/jobs, compare Id's to what's in events from `python-test-data-automation` events.

## todo
- see if anything else on `request.node.rep_call.` can be useful for debugging
- check `nohup.out` file for output of what went wrong, this may match the events captured by sdk, in `python-test-data-automation` project.
- breadcrumbs in events in `python-test-data-automation` reveal which `saucelabs.com` endpoints were failing (non-200 responses)
- compare to 'invalid selector css' errors in Saucelabs VM, are those relevant still? Why doesn't it find the div button?

## Conclusion
- Many possible points of failure, still uncertain what the probably cause was.
- The only info about why Saucelabs VM may have failed, is what SauceLabs sends back to you, which you can log in the pytest. i.e. we don't have an inside look of what's going on inside Saucelab's apps/cloud/VM's, only what they choose to report back to us.
- There are only so many places in the Pytest/selenium tests where an error status regarding a job can be indicated. This PR instrumented sentry_sdk to capture when that is happening. Still need more research on how to coordinate this data against what you can see in Saucelabs

image

### CODE REVIEW

File: nohup.out [hunk 1]
```diff
@@ -0,0 +1 @@
+nohup: ./script.sh: Permission denied
```
Comment (ndmanvar) [inline nohup.out:1]: @thinkocapo is this file necessary?

**Comment by ndmanvar** [comment]
👍 looks good. left one minor comment @thinkocapo

File: nohup.out (hunk 1 above)
Comment (thinkocapo) [reply nohup.out]: good catch, I've removed it.

The nohup.out file gets generated and captures all the output of the script and while loop, it's good for debugging, if needed. @ndmanvar
